conda/recipes/cucim/meta.yaml

```diff
@@ -43,7 +43,7 @@ requirements:
     - cudatoolkit ={{ cuda_version }}
     - cupy >=10,<12.0.0a0
     - libcucim ={{ version }}
-    - numpy >=1.21.3
+    - numpy =1.21.3
```

Comment:
Wonder if we should relax the patch version (as that shouldn't matter as much)

Reply:
The changelog for numpy 1.21.3 says it is the first with Python 3.10 support. I am fine with the relaxed pinning if that is not an issue.

Reply:
after committing this change, it looks like the `3.10` packages built with `numpy` version `1.21.6`, which should be fine. 

merging!